.changeset/calm-pants-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": patch
+---
+
+[Fixed] - fixed card responsive behaviours using css grid
```

Comment:
nitpick: do we need to repeat fix here?

Reply:
sorry probably not but I'll merge anyway if thats ok? its just taken a while to get this out